test/evergreen.yml

```diff
@@ -1384,6 +1394,23 @@ tasks:
               ./t -1 -C "direct_io=[data]" -c ../../../test/format/CONFIG.stress
             done
 
+  - name: test-checkpoint
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ CFLAGS="-ggdb"
```

Comment:
Is there any special consideration to use `clang` here? 
It would be good to add a comment to declare the reason why not enabling diagnostic - similar as what's being done in the current Jenkins shell.

Reply:
> Do you have a patch build with a run of the newly added `test-checkpoint` task?
> Alternatively, you can schedule the new task in the pull request triggered patch build.

https://evergreen.mongodb.com/version/5dc0c42f3627e02ee6bf7ca9 is the patch build for the newly added `test-checkpoint` task.

Reply:
Thanks.

Reply:
The diagnostic is not enabled in the Jenkins job as well. Added the comment for not including diagnostic.